e2e/basics.spec.js

```diff
@@ -146,6 +153,23 @@ describe('Basic React Native and Interaction Support', () => {
       assert(res.length).equal(1);
       assert(res[0]['i']).equal('foobar');
     });
+
+    it('should reset identity', async () => {
+      const { err1, err2, res1, res2 } = await new Promise(resolve => {
+        testUtil.findEventInRedisRequests({ t:  'pressInTestEvent2'}, (err1, res1) => {
```

Comment:
It's a little hard to follow what's happening here... Certainly, there's a lower threshold for readability, since this is test code, but perhaps factoring out a `findEventAfterSentinel` method or something would make it clearer?

Reply:
Could you elaborate on what you mean by `findEventAfterSentinel` and what you'd like pulled out here?

Reply:
I just mean that you have to jump through a couple of hoops to get what you want here, which makes this section particular hard to grok for the uninitiated.  It seemed to me like what you were really saying was "find this event after we're sure we've seen the sentinel".  Pulling that out into a named function might make this a lot easier to read for someone who's new to the project.

Reply:
All of these assertions happen after the sentinel anyway (since the `before` hook blocks until we see the sentinel).  I assume you mean before+after the `resetIdentity` call?

I don't think it's worth pulling this into a separate function.  Added comments for clarifying what this is doing